app/scripts.babel/contentscript.js

```diff
@@ -84,10 +84,12 @@ function toggle_columns() {
 
 function hide_columns() {
   chrome.storage.sync.get({ 'columnsToHide': '' }, function(items) {
-    let column_collection = items.columnsToHide.split(',')
-      .map(function(i) { return i.trim(); });
-    let css = generate_column_hiding_css(column_collection);
-    window.v1_column_hiding_style_el = add_css_to_document(css);
+    if (typeof items.columnsToHide != 'undefined') {
```

Comment:
@BaseCase Agreed. Reverted those changes.

Also, you should try the version on the Chrome Web Store: https://chrome.google.com/webstore/detail/versionone-shot/ifljgacfkdomobnaeibimidndmblihoh